impl/src/main/java/org/jboss/cdi/tck/tests/full/decorators/broken/toomanydelegateinjectionpoints/AfterBeanDiscoveryObserver.java

```diff
@@ -14,7 +14,7 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-package org.jboss.cdi.tck.tests.decorators.custom.broken.toomanydelegateinjectionpoints;
+package org.jboss.cdi.tck.tests.full.decorators.broken.toomanydelegateinjectionpoints;
```

Comment:
Should be `full.decorators.custom.broken.toomanydelegateinjectionpoints`, not `full.decorators.broken.toomanydelegateinjectionpoints`.

Reply:
Changed.